styles/project-viewer.less

```diff
@@ -137,6 +137,10 @@ project-viewer {
 	span.icon {
 		font-family: @font-family;
 	}
+
+	span.icon::before {
+		font-size: 16px;
```

Comment:
You're right. I was going off the [built-in size for Octicons](https://github.com/atom/atom/blob/master/static/variables/octicon-mixins.less). So I've done one better. In my update, the `16px` is the default and gets overridden by `@component-icon-size` as long as that variable has a valid value. If the user breaks their syntax theme or installs a bad one, the Project Viewer icons will still line up.